docs/prerequisites.adoc

```diff
@@ -50,16 +38,18 @@ If you are using Oracle Linux, you can install Terraform as follows:
 yum -y install oraclelinux-developer-release-el7 && yum -y install terraform
 ----
 
-==== Installing Terraform on macOS
+=== Installing Terraform on macOS
 
 If you are using a Mac, you can install Terraform as follows:
 
 [source,bash]
 ----
-brew install terraform
+brew install hashicorp/tap/terraform
 ----
 
-==== Manual Installation
+This will install Terraform using the official Hashicorp homebrew tap. For more details between the differences and benefits to use the community maintained or the official homebrew tap, see {uri-hashicorp-brew-tap-announcement}[Hashicorp's announcement]
```

Comment:
Homebrew is a product name and should be capitalised. HashiCorp should be camed-cased.